frontend/pages/login.tsx

```diff
@@ -72,18 +72,19 @@ const Login = () => {
           if (user.role === UserRole.Disabled) {
             router.push('/wait');
           } else {
-            // TODO this is a temporary fix
             const response = await axios.get<Edition>(Endpoints.EDITIONACTIVE, {
               headers: { Authorization: `Basic ${accessToken}` },
             });
-            if (response) {
+            if (response.data) {
               const editionName = response.data.name;
               setEdition(editionName);
               if (typeof window !== 'undefined' && editionName) {
                 localStorage.setItem('edition', editionName);
               }
+              router.push(`/${editionName}/projects`);
+            } else {
+              router.push('/editions');
```

Comment:
Only admins have access to the /editions page, so you should check ```user.role === UserRole.Admin``` there is a protection on the editions page that would just push a non-admin back to login page. 
Also you should use Endpoints.EDITIONS and Endpoints.PROJECTS instead of /projects and /editions
If there is no active edition I'm not sure where a non-admin user should be referred to, I don't think we have a page for that yet.

Reply:
Done. I think it makes sense to redirect coaches to the wait page when there is no active edition, so that's what I've implemented.